experiments/context_gating/run_context_gating.py

```diff
@@ -14,6 +14,30 @@
 from experiments.context_gating.utils import set_seed_everywhere
 from omegaconf import DictConfig, OmegaConf
 
+from carl.context_encoders import *
+import torch as th
+
+import pdb
+
+def get_encoder(cfg) -> ContextEncoder:
+    """
+    Loads the state dict of an already trained autoencoder.
+    """
+    model = None
+
+    if cfg.encoder.weights is not None:
+        encoder_cls  = eval(cfg.encoder.model)
```

Comment:
- [ ] you can import `import carl.context_encoders` and then use `getattr(carl.context_encoders, cfg.encoder.model)`

Reply:
Not required anymore; Shifted to saving and loading full models instead of state_dicts. Check commit 8ec875b8fab83265667ea7d55e1b66ca6e85e1e2